### PR TITLE
Add a guide for preparing the Rust annual survey

### PR DESCRIPTION
**Pull request by Kobzol**
A simple document that writes down what we have done this year to prepare the survey.

### CODE REVIEW

File: annual-survey.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+   - You can ask around on the `#rust-survey` or `#general` Zulip streams for assistance.
+7. Double-check the survey flow in SurveyHero
+   - Go through all the possible paths through the survey, and check that the questions and answers in SurveyHero are
+   the same as in `questions.md`.
+   - Possible problems:
+     - Missing answers or questions
+     - Wrong survey flow (questions being skipped, or not being skipped, when they should be)
+     - Radio buttons instead of checkboxes for exclusive answers
+     - Typos
+   - Optionally, the [`verifier`](verifier) binary can also be used to check this, if you are able to get access to the
```
Comment (apiraino) [inline annual-survey.md:31]: I think you wanted to add a markdown link to this `verifier`?

Comment (Kobzol) [reply annual-survey.md:31]: It is a markdown link, right? The file is in the root and points to the `verifier` directory.

File: annual-survey.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   - Ideally, ask in the [#rust-survey](https://rust-lang.zulipchat.com/#narrow/stream/402479-t-community.2Frust-survey) or
+   [#general](https://rust-lang.zulipchat.com/#narrow/stream/122651-general) Zulip streams, and also try to ask the individual
+   top level teams (`t-compiler`, `t-lang`, `t-cargo` etc.).
+4. Finalize the questions/answers for the given year (ideally by **end of September**)
+5. Put the survey questions to SurveyHero
+   - This needs collaboration with the Rust foundation, please ask in the [#rust-survey](https://rust-lang.zulipchat.com/#narrow/stream/402479-t-community.2Frust-survey)
+   Zulip stream for assistance.
+6. Find translator volunteers
+   - Either for translating the survey, or for reviewing auto-generated translations.
+   - You can ask around on the `#rust-survey` or `#general` Zulip streams for assistance.
```
Comment (apiraino) [inline annual-survey.md:22]: I'd expand a bit this section to mention the hidden traps in the translation workflow using SH (feel free to improve the wording)

```suggestion
6. Translate the questionnaire
   - SurveyHero can generate the questionnaire in other languages, uses some machine translation which often leaves a bit to desire
   - Find translators for reviewing the survey and the auto-generated translations
   - You can ask around on the `#rust-survey` or `#general` Zulip streams for assistance.
   - Be aware that the SurveyHero export feature is currently broken (breaks non Latin alphabets) and importing the translations is not allowed. Correcting the translations is therefore a manual work both for the translator and for the operator that has access to the SurveyHero backoffice.
   - Translators can take advantage of [this small guide](https://rust-lang.zulipchat.com/#narrow/stream/402479-t-community.2Frust-survey/topic/Translation.20guide/near/406836813)
```

**Review by apiraino — CHANGES_REQUESTED** [review]
had a quick read, that's all I can think of right now :slightly_smiling_face:

File: annual-survey.md (hunk 1 above)
Comment (apiraino) [reply annual-survey.md:31]: ah yes sorry I didnt check :smile:

**Comment by Kobzol** [comment]
Thanks! :) I updated the translation guide according to your suggestion.

**Pull request merged**